### PR TITLE
Update for compatibility with SDK v0.3.1

### PR DESCRIPTION
**Pull request by jmrossy**
- Update for compatibility with SDK v0.3.0+
- Remove dead package deps
- Rename `sdk` folder to `app`

### CODE REVIEW

File: src/test/deploy.test.ts [hunk 1]
```diff
@@ -22,24 +24,21 @@ describe('deploy', async () => {
...
+    const coreDeployer = new TestCoreDeployer(multiProvider);
     const coreContractsMaps = await coreDeployer.deploy();
     core = new TestCoreApp(coreContractsMaps, multiProvider);
     config = core.extendWithConnectionManagers(getConfigMap(signer.address));
     deployer = new HelloWorldDeployer(multiProvider, config, core);
   });
 
   it('deploys', async () => {
-    contracts = await deployer.deploy();
+    contracts = await deployer.deploy({});
```
Comment (jmrossy) [inline src/test/deploy.test.ts:37]: Ignore these deploy `{}` params, due to this line: https://github.com/abacus-network/abacus-monorepo/blob/main/typescript/deploy/src/router/deploy.ts#L94

Will fix in next 0.3.0 publish

File: contracts/HelloWorld.sol [hunk 2]
```diff
@@ -34,6 +34,11 @@ contract HelloWorld is Router {
         string message
     );
 
+    constructor() {
+        // Transfer ownership of the contract to deployer
+        _transferOwnership(msg.sender);
```
[the remark this answers is not in the document]
Comment (jmrossy) [reply contracts/HelloWorld.sol:39]: @yorhodes Would the fallback implementation you mentioned handle this? Or is the setting of ACM/IGP something we're trying to highlight here?

[the remark this answers is not in the document]
Comment (yorhodes) [reply contracts/HelloWorld.sol:39]: the fallback does handle this but might as well be explicit here if you already have a constructor

**Review by yorhodes — APPROVED** [review]
great!

**Pull request merged**